### PR TITLE
Propose banner at top of templates

### PR DESCRIPTION
**Pull request by miniksa**
Getting tired of obvious low quality issues and I want to provide the warning that we may start closing things without further explanation as our volume is too high to deal with junk issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ assignees: ''
...
 ---
 
+<!-- 
+🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
+
+I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
+1. If I delete this entire template and go my own path, the core team may close my issue without further explanation or engagement.
+2. If I list multiple bugs/concerns in this one issue, the core team may close my issue without further explanation or engagement.
+3. If I write an issue that has many duplicates, the core team may close my issue without further explanation or engagement (and without necessarily spending time to find the exact duplicate ID number).
+
```
Comment (miniksa) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: 4. If I leave the title incomplete when filing the issue, the core team may close my issue without further explanation or engagement.

Comment (miniksa) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: 5. If I file something completely blank in the body, the core team may close my issue without further explanation or engagement.

**Comment by DHowett-MSFT** [comment]
@yodurr has opinions here.

**Comment by miniksa** [comment]
Yeah yeah I'll wait. I just thought it food for discussion

**Pull request merged**